README.md

```diff
@@ -1,65 +1,107 @@
-<a href="https://github.com/TryGhost/Ghost"><img src="https://cloud.githubusercontent.com/assets/120485/18661790/cf942eda-7f17-11e6-9eb6-9c65bfc2abd8.png" alt="Ghost" /></a>
-<a href="https://travis-ci.org/TryGhost/Ghost"><img align="right" src="https://travis-ci.org/TryGhost/Ghost.svg?branch=master" alt="Build status" /></a>
+&nbsp;
+<p align="center">
+  <a href="https://ghost.org">
+    <img src="https://user-images.githubusercontent.com/120485/43974508-b64b2fe8-9cd2-11e8-8e58-707254b8817c.png" width="200px" alt="Ghost" />
+  </a>
+</p>
+<h3 align="center">Fiercely independent, professional publishing</h3>
+<p align="center">A fully open source, powerful platform for building and running modern publications,<br>
+    we power serious blogs, magazines and journalism from DuckDuckGo to OpenAI & Sky News.</p>
+<hr />
+<p align="center">
+    <a href="https://ghost.org">Ghost.org</a> |
+    <a href="https://ghost.org/features">Features</a> |
+    <a href="https://ghost.org/customers">Showcase</a> |
+    <a href="https://forum.ghost.org">Forum</a> |
+    <a href="https://docs.ghost.org">Documentation</a> |
+    <a href="https://docs.ghost.org/v1/docs/contributing">Contributing</a> |
```

Comment:
We will merge this PR into the Ghost 2.0 branch. 
All links in master point already to the v1 docs.

This link should not contain `/v1`/.